src/data/shields.ts

```diff
@@ -3,9 +3,9 @@ export const shields = {
   basic: {heater: 12, spanish: 6, french: 1},
   regional: {horsehead: 1, horsehead2: 1, polish: 1, hessen: 1, swiss: 1},
   historical: {boeotian: 1, roman: 2, kite: 1, oldFrench: 5, renaissance: 2, baroque: 2},
-  specific: {targe: 1, targe2: 0, pavise: 5, wedged: 10},
+  specific: {targe: 1, targe2: 0, pavise: 5, wedged: 10, embowed: 1},
   banner: {flag: 1, pennon: 0, guidon: 0, banner: 0, dovetail: 1, gonfalon: 5, pennant: 0},
-  simple: {round: 12, oval: 6, vesicaPiscis: 1, square: 1, diamond: 2, no: 0},
+  simple: {round: 12, oval: 6, vesicaPiscis: 1, square: 1, diamond: 2, hexagon: 0, no: 0},
```

Comment:
Should it be at least `1`?

Reply:
I was worried that you wouldn't want it to appear because it's definitely a non-historical shape.

Reply:
That's fine